wallet/how-to/interact-with-smart-contracts/unity/contract-interface.md

```diff
@@ -0,0 +1,150 @@
+---
+sidebar_position: 1
+---
+
+# Contract interface
+
+When [interacting with smart contracts in Unity](index.md), the `IContract` interface defines an
+interface of a given contract.
+This interface includes all the functions of the contract, and optionally includes a constructor
+function and `Bytecode` field.
+
+You can use the [contract code generator](index.md#generate-contract-code) to generate a contract
+interface given a contract ABI or Hardhat artifact JSON file.
+
+To manually create a new contract interface, you must define a new interface that inherits from `IContract`.
+You may optionally declare the [`BackedType` attribute](contract-factory.md#backed-type-contract-factory).
```

Comment:
Replace "may" with "can"? For normalization, structure, and minimalization purposes, you may just want "Optionally, declare the..."